src/Files.App/Views/LayoutModes/BaseLayout.cs

```diff
@@ -901,7 +961,10 @@ protected void FileList_DragItemsStarting(object sender, DragItemsStartingEventA
 			try
 			{
 				var shellItemList = e.Items.OfType<ListedItem>().Select(x => new VanaraWindowsShell.ShellItem(x.ItemPath)).ToArray();
-				if (shellItemList[0].FileSystemPath is not null && !InstanceViewModel.IsPageTypeSearchResults)
+
+				if (shellItemList[0].FileSystemPath is not null &&
+					InstanceViewModel is not null && 
+					!InstanceViewModel.IsPageTypeSearchResults)
```

Comment:
`!` is redundant with the `null` test the line above?

Reply:
You're recommending `!InstanceViewModel is null`? I don't prefer to use `!`, `==`, and `!=` for null check.